src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -806,6 +807,7 @@ private void DeleteGeometries(KeyValuePair<string, Model3D>[] geometryModels, bo
                     if (model3D != null)
                     {
                         model3D.Detach();
+                        model3D.Dispose();
```

Comment:
when we remove geometry from the background preview or watch3d node - actually dispose the tessellated objects, don't just detach them.